test/e2e/cloud/gcp/cluster_upgrade.go

```diff
@@ -279,6 +283,72 @@ var _ = SIGDescribe("gpu Upgrade [Feature:GPUUpgrade]", func() {
 	})
 })
 
+var _ = SIGDescribe("TopologyManager Upgrade [Feature:TopologyManagerUpgrade]", func() {
```

Comment:
At this time, test-infra contains just `"\[Feature:TopologyManager\]"` without `Upgrade` keyword in sig-node/node-kubelet.yaml and sig-node/sig-node-presubmit.yaml.
Will you add the above keyword also? Or should we change here to `"\[Feature:TopologyManager\]"` to run this test on the above test jobs?

Reply:
Will sync with the node team and figure out, I don't have an answer for this.

Reply:
Hi, partial followup, still figuring out the path forward with the other parties.
I was trying to make my code work and eventually checked out how the existing code is working, but according to https://testgrid.k8s.io/google-gce-upgrade even the existing tests are red, and the last upgrade check configured was from 1.13 and 1.14. This was unexpected, am I looking in the wrong place perhaps? If so what's the right place to check how existing test are doing?
If the existing tests are indeed red the simplest way forward is drop this PR and restart from scratch in the node area.
Thanks for any help.